DEPLOYMENT.md

```diff
@@ -0,0 +1,57 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wild card SSL certificate be installed on the OpenShift router. However, in situations where this is not desirable, RHMAP can be exposed using single URL.
+
+## RHMAP Proxy
```

Comment:
Hmm yeah I just took the name from @maleck13 doc but I'm easy on this one. My suggestion on irc was more so that whatever we choose we shuld keep the names consistent. The dc is currently called nginx-proxy and I think we should sync all these up to avoid confusion